cldrDict_sconscript

```diff
@@ -80,18 +80,18 @@ NVDAToCLDRLocales = {
 	"it":("it",),
 	"ja":("ja",),
 	"ka":("ka",),
-	#"kmr":(),
+	#"kmr":("ku",),
```

Comment:
should this still be commented?

Reply:
Oh I see that it was failing to build. If we are unable to include it, this line should be unchanged in the diff. 
```suggestion
	#"kmr":(),
```

Reply:
This line will be included with the comment mark when CLDR files are updated. Even if we do not leave it as it is, it is enough to close this line with a comment mark.

Reply:
I'd also add that the comment should include "why" it is commented out. EG it fails to build with error XYZ